Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Storefront.m

```diff
@@ -124,6 +124,21 @@ - (NSOperation *)getProductsByIds:(NSArray *)productIds completion:(BUYDataProdu
 	}];
 }
 
+- (NSOperation *)getProductsByTags:(NSArray<NSString *> *)tags completion:(BUYDataProductsBlock)block
+{
+	NSURL *url = [self urlForProductListingsWithParameters:@{
+															 @"tag": [tags componentsJoinedByString:@","],
+															 }];
+	return [self getRequestForURL:url completionHandler:^(NSDictionary *json, NSHTTPURLResponse *response, NSError *error) {
+		
+		NSArray *products;
+		if (json && !error) {
+			products = [self.modelManager insertProductsWithJSONArray:json[BUYProductsKey]];
+		}
```

Comment:
can the response be extracted to avoid duplication from the other methods returning products?